src/Components/App/App.js

```diff
@@ -12,41 +11,42 @@ class App extends Component {
     super()
     this.state = {
       movies: [],
-      clicked: false
+      clicked: false,
+      singleMovieDetails: [],
+      err: ''
```

Comment:
I like that you're using the empty data states as your initial values here (instead of using 'null').  This shows me at-a-glance what shape that data will eventually take.